src/Common/src/System/PasteArguments.cs

```diff
@@ -0,0 +1,144 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Linq;
```

Comment:
You've gotten rid of the Linq dependency in the code - the namespace import shouldgo too or it won't go into S.R.Extensions.